config/development.js

```diff
@@ -3,6 +3,9 @@ exports.config = {
   common: {
     database: {
       name: process.env.DB_NAME_DEV
+    },
+    numbers_api: {
+      url: process.env.NUMBERS_API_URL
```

Comment:
add this new env var to the file `.env.example ` so when a new dev gets in the project is easy to him to know what env vars are missing.